src/components/Book.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prop-types */
```

Comment:
Kindly remove all disabled Linters, Linting helps in error-free and professional code 👍 



Please  check for other occurrences 🙂